vividus-plugin-web-app/src/main/resources/steps/defaults/composite.steps

```diff
@@ -98,3 +98,81 @@ When I enter `<text>` in field located `By.xpath(<xpath>)`
 
 Composite: When I select '$text' from a drop down by the xpath '$xpath'
 When I select `$text` from drop down located `By.xpath(<xpath>)`
+
+Composite: Then an element with the name '$elementName' containing text '$text' exists
+Then number of elements found by `By.elementName(<elementName>):a->filter.textPart(<text>)` is equal to `1`
```

Comment:
Should we realy check visible and invisible elements ?
Cause I see that default value defined in code for  Visibility =VISIBLE. So maybe it is overhead ?